### ISSUE TITLE
Standardise fonts

### ISSUE THREAD

**Issue by naastik** [shadowcatx/shadowcatx.github.io#6]
Lots of different fonts seem to be in use in the "Getting Started" section. Using fewer and consistent fonts will make the section more readable

**Comment by shadowcatx** [comment]
Yeah I think Jan just placed them there as placeholder, which is why he was asking me to style these items yesterday. But it's only available to me when you accepted his change request 4 minutes earlier. 

If we are to have this kind of pull request and changes based on the requests, shall we still have some kind of dev branch?

**Comment by naastik** [comment]
The website doesn't really have a build/deploy plan so I think master + feature branches, PR's should do. Fork repo, make changes, test locally, if happy with local changes, submit PR.
